scripts/apply.ts

```diff
@@ -0,0 +1,96 @@
+import fs from "fs";
```

Comment:
```suggestion
import fs from "fs/promises";
```
try this to avoid promisify 🙂